### PR TITLE
feat(tiamu): uniform code (#11)

### PR DESCRIPTION
**Pull request by duanfuxiang0**
## Summary about this PR

Issue Number: close #issue_number_you_created

## Tests Check List

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [x] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: storage/tianmu/mm/huge_heap_policy.cpp [hunk 1]
```diff
@@ -62,7 +62,7 @@ HugeHeap::HugeHeap(std::string hugedir, size_t size) : TCMHeap(0) {
     rc_control_ << system::lock << "Huge Heap size (MB) " << (int)(size) << system::unlock;
     // size_ = size;
     // manage the region as a normal 4k pagesize heap
-    m_heap.RegisterArea(heap_frame_, size_ >> kPageShift);
+    m_heap_.RegisterArea(heap_frame_, size_ >> kPageShift);
```
Comment (hustjieke) [inline storage/tianmu/mm/huge_heap_policy.cpp]: `m_heap_` --> `heap_`, the prefix `m_` is another naming style, here we follow google C++ style.

File: storage/tianmu/mm/memory_handling_policy.h [hunk 2]
```diff
@@ -39,22 +39,23 @@ class MemoryHandling {
...
   using PtrHeapMap = std::unordered_map<void *, HeapPolicy *>;
-  std::unordered_map<TraceableObject *, PtrHeapMap *> m_objs;
+  std::unordered_map<TraceableObject *, PtrHeapMap *> m_objs_;
 
-  HeapPolicy *m_main_heap, *m_huge_heap, *m_system, *m_large_temp;
-  int main_heap_MB, comp_heap_MB;
-  bool m_hard_limit;
-  int m_release_count = 0;
-  size_t m_release_total = 0;
+  HeapPolicy *m_main_heap_, *m_huge_heap_, *m_system_, *m_large_temp_;
```
Comment (hustjieke) [inline storage/tianmu/mm/memory_handling_policy.h]: Fix  prefix `m_` with above.

File: storage/tianmu/mm/memory_handling_policy.h [hunk 3]
```diff
@@ -39,22 +39,23 @@ class MemoryHandling {
...
+  int main_heap_size_, comp_heap_size_;  // MB
+  bool m_hard_limit_;
```
Comment (hustjieke) [inline storage/tianmu/mm/memory_handling_policy.h]: Do init in class definition just like other variables does.

File: storage/tianmu/system/tianmu_file.cpp [hunk 4]
```diff
@@ -52,23 +54,23 @@ int TianmuFile::Flush() {
 }
 
 int TianmuFile::OpenCreate(std::string const &file) {
-  return Open(file, O_CREAT | O_RDWR | O_LARGEFILE | O_BINARY, tianmu_umask);
+  return Open(file, O_CREAT | O_RDWR | O_LARGEFILE | O_BINARY, TIANMU_UMASK);
```
Comment (hustjieke) [inline storage/tianmu/system/tianmu_file.cpp:57]: Is `TIANMU_UMASK` a constant name? or an enum? Fix it with google style:
https://google.github.io/styleguide/cppguide.html#Constant_Names

**Review by hustjieke — CHANGES_REQUESTED** [review]

File: storage/tianmu/system/tianmu_file.cpp (hunk 4 above)
Comment (duanfuxiang0) [reply storage/tianmu/system/tianmu_file.cpp:57]: is Macro

File: storage/tianmu/mm/memory_handling_policy.h (hunk 3 above)
Comment (duanfuxiang0) [reply storage/tianmu/mm/memory_handling_policy.h]: ok

**Review by RingsC — APPROVED** [review]
LGTM

**Review by hustjieke — APPROVED** [review]
LGTM

**Comment by hustjieke** [comment]
Please resove conflict, Ping @duanfuxiang0

**Comment by hustjieke** [comment]
No deed any more, marked as deprecated!

**Pull request closed without merging**